metricflow/aggregation_properties.py

```diff
@@ -17,18 +17,29 @@ class AggregationType(ExtendedEnum):
     SUM_BOOLEAN = "sum_boolean"
     AVERAGE = "average"
 
-    # COUNT = "count" not yet implemented ... non-expansive as COUNT(COUNT(X), COUNT(Y)) != COUNT(COUNT(X ∪ Y))
+    COUNT = "count"
     # AVERAGE = "average"  not yet implemented ...requires us to keep track of two quantities, count and sum
 
     @property
     def is_expansive(self) -> bool:
         """Expansive ≝ Op( X ∪ Y ∪ ...) = Op( Op(X) ∪ Op(Y) ∪ ...)"""
-        return self in (AggregationType.SUM, AggregationType.MIN, AggregationType.MAX, AggregationType.BOOLEAN)
+        return self in (
+            AggregationType.SUM,
+            AggregationType.MIN,
+            AggregationType.MAX,
+            AggregationType.BOOLEAN,
+            AggregationType.COUNT,
```

Comment:
Can you update the docstring to indicate that COUNT is expansive because it gets transformed into SUM prior to initial aggregation?